src/ResponseTagger.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache;
+
+use FOS\HttpCache\ProxyClient\Invalidation\TagsInterface;
+
+/**
+ * Service for Response cache tagging.
+ *
+ * @author David de Boer <david@driebit.nl>
+ * @author David Buchmann <mail@davidbu.ch>
+ * @author André Rømcke <ar@ez.no>
+ */
+class ResponseTagger
+{
+    /**
+     * @var TagsInterface
+     */
+    private $client;
+
+    /**
+     * @var array
+     */
+    private $tags = [];
+
+    /**
+     * Constructor
+     *
+     * @param TagsInterface $client
+     */
+    public function __construct(TagsInterface $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * Get the HTTP header name that will hold cache tags.
+     *
+     * @return string
+     */
+    public function getTagsHeaderName()
+    {
+        return $this->client->getTagsHeaderName();
```

Comment:
*Continuing discussion from https://github.com/FriendsOfSymfony/FOSHttpCache/pull/237#discussion_r45050050*

>> Do we need to forward this call to the client?
> the logic of moving it to client was that only client knows how it needs to be escaped for specifics on proxy

However maybe this is not needed, constraints I know about:
- For Sf I was kind of thinking the header name needs to be hard coded, but I guess it does not have to do that
- For Nginx I have yet to look into which plugins would make sense to enable this, and which constraints they will enforce

So maybe this can be fully handled in ResponseTagger, `escapeTags ` is rather generic anyway.
However be aware both Proxy and ResponseTagger will need to know about header then, but not a blocker, can use same param in bundle.